pyproject.toml

```
@@ -15,8 +15,9 @@ license = {text = "Apache-2"}
 classifiers = [
 ]
 dependencies = [
-    "notify2",
-    "psutil"
+    "psutil",
+    "pygobject",
+    "dbus-python"
```

You can use trailing comas in toml files
```suggestion
    "dbus-python",
```